apps/testingaccessibility/src/server/get-purchased-product.ts

```diff
@@ -37,9 +37,9 @@ export async function getPurchasedProduct(
   const token = await getToken({req})
   if (token && token.sub) {
     // no need to reload purchases since they are on the session
-    const purchases = token.purchases
+    const purchases = (token.purchases as any[]) || []
 
-    if (isArray(purchases)) {
+    if (!isEmpty(purchases)) {
```

Comment:
`isEmpty` will see if the thing passed in has stuff, which is more valuable here than if it's an array or not because that can (often is in this case) empty for various reasons